homeassistant/components/tautulli/sensor.py

```diff
@@ -82,62 +266,64 @@ async def async_setup_entry(
 ) -> None:
     """Set up Tautulli sensor."""
     coordinator: TautulliDataUpdateCoordinator = hass.data[DOMAIN]
-    async_add_entities(
+    entities: list[TautulliSensor | TautulliSessionSensor] = [
         TautulliSensor(
             coordinator,
             description,
         )
         for description in SENSOR_TYPES
-    )
+    ]
+    if coordinator.users:
+        entities.extend(
+            TautulliSessionSensor(
+                coordinator,
+                description,
+                user,
+            )
+            for description in SESSION_SENSOR_TYPES
+            for user in coordinator.users
+            if user.username != "Local"
+        )
+    async_add_entities(entities)
 
 
 class TautulliSensor(TautulliEntity, SensorEntity):
     """Representation of a Tautulli sensor."""
 
+    entity_description: TautulliSensorEntityDescription
+
     @property
-    def native_value(self) -> StateType:
+    def native_value(self) -> Any:
```

Comment:
This change should not be needed, what part/value did it complain about?

Reply:
And that is not because of `value_fn: Callable[[PyTautulliApiHomeStats, PyTautulliApiActivity, str], Any]` ?

Reply:
Apparently